### DISCUSSION TITLE
PVHI (Photovoltaic Heat Island) Computation for solarfarm development

### DISCUSSION THREAD

**Discussion by LordMicidial** [UMEP-dev/UMEP#643] [Q&A] [answered]
Dear all, 

I am an engineer passionate about mathematical models applied to the environmental field. Actually I work for the nascent photovoltaic industry, agrivoltaics to be picky.

In the course of the permitting process for a agrivoltaic plant (20 MW) we would like to build, the public agency asked me to build a mathematical model using a list of possible models including UMEP (SOLWEIG). So I am here in the dedicated forum.

Doing several searches, on the forum and online, I found Dr. Bartesaghi tutorials that I found extremely interesting. However I see no specific tutorial on the possibilities that such qGIS plug-in could have when it comes to PVHI. 

However, I am quite skeptical that there is anything (reliable) in the literature at the moment and that it can be done easily for us practitioners - I also see on your official site, the only tutorial on a UHI (https://umep-docs.readthedocs.io/projects/tutorial/en/latest/Tutorials/UWGSpatial.html#uwgspatial). 

Even if assuming that the same tutorial can be used to compute PVHI, I see that the tutorial is not ready. In fact, I read, “NOT READY BEYOND THIS POINT” in the middle of the web page.

Can you tell me whether the tutorial is complete and reliable and in principle what you think with respect to the reliability of UMEP to predict the PVHI of a utility-scale agrivoltaic plant?

Since I have not developed the plant I only have the DTM of a farmland. Therefore, I should somehow generate a DTM post implementation of the plant. It seems to me that no one has ever done such computations with UMEP. Do you confirm?

**Comment by biglimp** [comment, marked as answer]
Hi,
Interesting task you want to accomplish.
Most probably "heat islands" from solar energy plant and cities will look very different, mainly because solar plants have much lower thermal mass and less human activity which is needed for a nocturnal UHI to develop. Daytime UHI is very week due to turbulent mixing so I would not consider this as a big issue. Nevertheless, regarding the tutorial you mention, we are working on completing this in the near future. We are also adding another UHI model (TARGET) that so will be available in UMEP.

Regarding your second question, yes, you should create a DSM where your solar panels are included. Than you can use SEBE to estimate solar energy potential.
